src/scripts/main.js

```diff
@@ -1,3 +1,135 @@
 'use strict';
 
-// write code here
+const tr = document.querySelectorAll('tr');
+const tbody = document.querySelector('tbody');
+
+for (let i = 1; i < tr.length - 1; i++) {
+  tr[i].className = 'data';
+}
+
+const dataTr = document.querySelectorAll('.data');
+const list = [...dataTr];
```

Comment:
```suggestion
const list = [...tr];
```
Зачем класс `.data`?